bindings/azure/cosmosdb/cosmosdb.go

```diff
@@ -7,10 +7,13 @@ package cosmosdb
 
 import (
 	"encoding/json"
+	"errors"
 	"fmt"
 	"strings"
+	"time"
 
 	"github.com/a8m/documentdb"
+	"github.com/cenkalti/backoff/v4"
```

Comment:
How about using `github.com/cenkalti/backoff` as introduced in `go.mod`?

Reply:
We are explicitly using `v4` in our code

e.g. https://github.com/dapr/kit/blob/main/retry/retry.go#L8